src/applications/ivc-champva/10-7959a/containers/IntroductionPage.jsx

```diff
@@ -28,9 +111,7 @@ const IntroductionPage = ({ route }) => {
     [customText, formId, pageList, prefillEnabled, savedFormMessages],
   );
 
-  useEffect(() => {
-    focusElement('.va-nav-breadcrumbs-list');
-  }, []);
+  useEffect(() => focusElement('.schemaform-intro h1'), []);
```

Comment:
This is targeting an `h1` but I only see an `h2` within the `schemaform-intro` block

Reply:
The H1 is rendered with the `<FormTitle />` component.